content/2020/2020-11/2020-11-20-pulsar-weekly.md

```diff
@@ -0,0 +1,143 @@
+
+- [Broker] Make the `ServerCnx` that is used by the producer and consumer independent from Netty.
+
+    https://github.com/apache/pulsar/pull/6720 (@[cbornet](https://github.com/cbornet))
+
+- [Schema Registry] Support decoding multi-version reader to ensure users custom the reader to decode multi-version messages.
+
+    https://github.com/apache/pulsar/pull/8464 (@[congbobo184](https://github.com/congbobo184))
+
+- [Transaction] Add the transaction integration test.
+
+    https://github.com/apache/pulsar/pull/8594 (@[gaoran10](https://github.com/gaoran10))
+
+- [Pulsar IO] Add the NSQ Source.
+
+    https://github.com/apache/pulsar/pull/8250 (@[flowchartsman](https://github.com/flowchartsman))
+
+- [Go Functions] Access to user configuration before the Function loop.
+
+    https://github.com/apache/pulsar/pull/8467 (@[flowchartsman](https://github.com/flowchartsman))
+
+- [Broker] Improve the performance of `checkPublishRate()`.
+
+    https://github.com/apache/pulsar/pull/8623 (@[wangjialing218](https://github.com/wangjialing218))
+
+- [Managed Ledger] Perform periodic flush of `ManagedCursor` mark-delete positions.
+
+    https://github.com/apache/pulsar/pull/8634/files (@[merlimat](https://github.com/merlimat))
+
+- [Transaction] Support sending transaction messages synchronously.
+
+    https://github.com/apache/pulsar/pull/8631 (@[gaoran10](https://github.com/gaoran10))
+
+- [Proxy] Support enabling WebSocket on the Pulsar Proxy.
+
+    https://github.com/apache/pulsar/pull/8613 (@[codelipenghui](https://github.com/codelipenghui))
+
+- [Function] Enable Pulsar Function to send message to external Pulsar clusters.
+
+    https://github.com/apache/pulsar/pull/8434 (@[nlu90](https://github.com/nlu90))
+
+### Notable Bug Fix
+
+- [C++] Fix flaky tests of `KeySharedConsumerTest`.
+
+    https://github.com/apache/pulsar/pull/8568 (@[BewareMyPower](https://github.com/BewareMyPower))
+
+- [Transaction] Fix the Github CI test.
+
+    https://github.com/apache/pulsar/pull/8541 (@[congbobo184](https://github.com/congbobo184))
+
+- [Broker] Fix the issue that the managed-ledger `addEntry-latency` bucket stats are measured per minute.
+
+    https://github.com/apache/pulsar/pull/8218 (@[rdhabalia](https://github.com/rdhabalia))
+
+- [Broker] Fix the incorrect add-entry latency.
+
+    https://github.com/apache/pulsar/pull/8219 (@[rdhabalia](https://github.com/rdhabalia))
+
+- [SQL] Fix the issue that Pulsar SQL cannot query the last message.
+
+    https://github.com/apache/pulsar/pull/8635 (@[zymap](https://github.com/zymap))
+
+### Event / News
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
+
+  - All video recordings are available at [here](https://streamnative.io/resource#tgip).
+
+- [Upcoming Event]
+
+  -  Pulsar Summit Asia 2020
+
+     https://pulsar-summit.org/en/event/asia-2020
+
+### Blog / Article
+
+- Pulsar Advantages Over Kafka -- Javier Ramos
```

Comment:
1. 2.6.2 release blog:
2. schedule announcement: https://streamnative.io/en/blog/community/2020-11-17-pulsar-summit-asia-2020-announcement
3. Benchmark and Tencent Angel whitepaper